src/main/java/com/estonianec/hw2dot4/services/EmployeeService.java

```diff
@@ -0,0 +1,15 @@
+package com.estonianec.hw2dot4.services;
+
+import com.estonianec.hw2dot4.exceptions.EmployeeArrayOverflowException;
+import com.estonianec.hw2dot4.exceptions.EmployeeNotFoundException;
+
+import java.util.List;
+
+public interface EmployeeService {
+
+    List showAllEmployees();
```

Comment:
Лучше всегда указывать тип обрабатываемых данных, так мы будем уверены, что коллекция содержит элементы нужного нам типа
```suggestion
    List<String> showAllEmployees();
```